generate_nibetaseries_images.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+set -e
+
+# Generate Dockerfile.
+generate_docker() {
+  docker run --rm kaczmarj/neurodocker:0.5.0 generate docker \
+    --base=neurodebian:stretch-non-free \
+    --install gcc g++ graphviz \
+    --pkg-manager=apt \
+    --user=neuro \
+    --miniconda \
+      conda_install="python=3.6" \
+      pip_install="jupyter seaborn tox" \
```

Comment:
I added `tox` to test the functionality of the code inside the container

Reply:
Sorry, I forgot about that.